map/framework.cpp

```diff
@@ -685,11 +686,15 @@ void Framework::RegisterAllMaps()
 
     MwmSet::MwmId const & id = p.first;
     ASSERT(id.IsAlive(), ());
-    minFormat = min(minFormat, static_cast<int>(id.GetInfo()->m_version.GetFormat()));
+    auto const format = id.GetInfo()->m_version.GetFormat();
+    minFormat = min(minFormat, static_cast<int>(format));
     if (needStatisticsUpdate)
     {
       listRegisteredMaps << localFile->GetCountryName() << ":" << id.GetInfo()->GetVersion() << ";";
     }
+
+    version::MwmTraits mwmTraits(format);
+    alohalytics::LogEvent("SearchIndexFormat", DebugPrint(mwmTraits.GetSearchIndexFormat()));
```

Comment:
Please, don't use DebugPrint() for aloha events! You need to implement your own serialization.